delta/app/src/test/resources/permissions/eventstream-0-5.txt

```diff
@@ -1,20 +1,20 @@
-data:{"@context":["https://bluebrain.github.io/nexus/contexts/metadata.json","https://bluebrain.github.io/nexus/contexts/permissions.json"],"@type":"PermissionsAppended","_instant":"1970-01-01T00:00:00Z","_rev":1,"_subject":"http://localhost/v1/anonymous","permissions":["acls/read"]}
+data:{"@context":["https://bluebrain.github.io/nexus/contexts/metadata.json","https://bluebrain.github.io/nexus/contexts/permissions.json"],"@id":"http://localhost/v1/permissions","@type":"PermissionsAppended","_instant":"1970-01-01T00:00:00Z","_rev":1,"_subject":"http://localhost/v1/anonymous","permissions":["acls/read"]}
```

Comment:
we've discussed this a long time ago, events should not have an `@id` field:

- the id is not that useful as there's no way to retrieve an event by id
- the id of the resource should not be the id of the event

In the case of "kg" resources we've used the `_resourceId` field to encode the corresponding resource. We should keep that if we want to specify the corresponding resource id.

Reply:
SSEs are anyway messed up in several other ways. I'm not sure if this is in the scope of the current PR. For some events we had `@id` (realms, acls, orgs, projects), for some we had `_resourceId` (resources, schemas, resolvers) and for some we had nothing (permissions).

I think this should be tacked in the jointly with the issue: https://github.com/BlueBrain/nexus/issues/1846